src/components/views/elements/SSOButtons.tsx

```diff
@@ -72,7 +73,7 @@ const SSOButton: React.FC<ISSOButtonProps> = ({
         brandClass = `mx_SSOButton_brand_${brandName}`;
         icon = <img src={brandIcon} height="24" width="24" alt={brandName} />;
     } else if (typeof idp?.icon === "string" && idp.icon.startsWith("mxc://")) {
-        const src = matrixClient.mxcUrlToHttp(idp.icon, 24, 24, "crop", true);
```

Comment:
It seems like the `allowDirectLinks` behaviour got lost here...?

Reply:
Ftr that is a backwards compatibility that is not needed going forward

Reply:
Yea, that's the sort of thing that was supposed to be dropped ages ago.

Reply:
Should it be removed from the JS SDK as well then?

Reply:
That would be more difficult as it could be considered a breaking change. I've at least opened an issue to track it: https://github.com/matrix-org/matrix-js-sdk/issues/1643